persistent/picklecache.py

```diff
@@ -336,7 +336,10 @@ def _sweep(self, target, target_size_bytes=0):
         i = -1
         to_eject = []
         for value in self.ring:
-            if self.non_ghost_count <= target and (self.total_estimated_size <= target_size_bytes or not target_size_bytes):
+            if ((target or target_size_bytes) and
+               (not target or self.non_ghost_count <= target) and
```

Comment:
Line breaks *before* the binary operators, please. (Per this [pep-8 update](https://github.com/python/peps/commit/7690a00d478866eb3cae3d82bbe1e80199a7392f)) (That is, `and` and `or` belong at the beginning of the line.) E.g.,
```python
if (thing 
     and other_thing
     or this_thing):
```